sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Release History
 
+## 1.2.1 (2023-07-20)
+
+### Features Added
+
+- New function `*ClientFactory.NewDppResourceGuardProxyClient() *DppResourceGuardProxyClient`
```

Comment:
Would prefer to have some description about the new features instead of actual code changes.